fares-data-build-tool/create-data
# feat: Removing ability to create carnet academic tickets

## Description

It has been deemed it is not suitable for users to be able to create carnet academic tickets, so that user journey has been removed. 2 pages were also obsolete after this change, so they have been removed.

## Testing instructions

N/A - UI tests will generate products for the user journeys, then we can export on test env to ensure no issues on the netex side.
